distribution/pom.xml

```diff
@@ -174,6 +83,111 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>dist</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>tar</name>
```

Comment:
Would you explain how this property is being used?

Reply:
Sure, we also can use `-Dtar` to execute this plugin. In addition, if we used both of `name` and `value` tags, then only `-Dtar=dist` will be in effect.

```xml
<property>  
    <name>tar</name>  
    <value>dist</value>  
</property>
```

Reply:
Okay.

Reply:
Added.